Bottom: 42f6a52ff440cd3f7f6054e7c9fdc8e16e7b262c
Top:    2eb7e1defb3686d48a461fc6223f01a49d3d3381
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-06 23:35:02 +0100

Refresh of introduce-polygonhole_mode-for

---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3805856..0db5aad 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -508,6 +508,7 @@ PopupMenus =
        {"Text" checked=textmode,1 Mode(Text) a={"F4" "<Key>F4"}}
        {"Rectangle" checked=rectanglemode,1 Mode(Rectangle) a={"F5" "<Key>F5"}}
        {"Polygon" checked=polygonmode,1 Mode(Polygon) a={"F6" "<Key>F6"}}
+       {"Polygon Hole" checked=polygonholemode,1 Mode(PolygonHole)}
        {"Buffer" checked=pastebuffermode,1 Mode(PasteBuffer) a={"F7" "<Key>F7"}}
        {"Remove" checked=removemode,1 Mode(Remove) a={"F8" "<Key>F8"}}
        {"Rotate" checked=rotatemode,1 Mode(Rotate) a={"F9" "<Key>F9"}}
@@ -518,7 +519,6 @@ PopupMenus =
        {"Copy" checked=copymode,1 Mode(Copy)}
        {"Lock" checked=lockmode,1 Mode(Lock) a={"F12" "<Key>F12"}}
        {"Cancel" Mode(Escape) a={"Esc" "<Key>Escape"}}
-       {"PolygonHole" checked=polygonholemode,1 Mode(PolygonHole)}
       }
     }
   }
diff --git a/src/pcb-menu.res b/src/pcb-menu.res
index 3f0ffa2..0df4b7f 100644
--- a/src/pcb-menu.res
+++ b/src/pcb-menu.res
@@ -174,6 +174,7 @@ MainMenu =
    {"Text" checked=textmode,1 Mode(Text) a={"F4" "<Key>F4"}}
    {"Rectangle" checked=rectanglemode,1 Mode(Rectangle) a={"F5" "<Key>F5"}}
    {"Polygon" checked=polygonmode,1 Mode(Polygon) a={"F6" "<Key>F6"}}
+   {"Polygon Hole" checked=polygonholemode,1 Mode(PolygonHole)}
    {"Buffer" checked=pastebuffermode,1 Mode(PasteBuffer) a={"F7" "<Key>F7"}}
    {"Remove" checked=removemode,1 Mode(Remove) a={"F8" "<Key>F8"}}
    {"Rotate" checked=rotatemode,1 Mode(Rotate) a={"F9" "<Key>F9"}}
@@ -184,7 +185,6 @@ MainMenu =
    {"Copy" checked=copymode,1 Mode(Copy)}
    {"Lock" checked=lockmode,1 Mode(Lock) a={"F12" "<Key>F12"}}
    {"Cancel" Mode(Cancel) a={"Esc" "<Key>Escape"}}
-   {"PolygonHole" checked=polygonholemode,1 Mode(PolygonHole)}
    -
    {"Command" Command() a={":" "<Key>:"}}
   }
